File: js/player.js
function Player() {
  Paddle.call(this);

  this.x = 20;
}

Player.prototype = Object.create(Paddle.prototype);
Player.prototype.constructor = Player;

Player.prototype.update = function() {
  var speed = 15;

  if (game.keyPressed.up) {
    this.yVelocity = -speed;
  } else if (game.keyPressed.down) {
    this.yVelocity = speed;
  } else {
    this.yVelocity = 0;
  }

  Paddle.prototype.update.apply(this, arguments);
};

File: app/models/player.rb
class Player < Paddle

  def initialize
    super
    @x = 20
  end

  def update
    speed = 15;

    if @game.key_pressed[:up]
      @y_velocity = -speed;
    elsif @game.key_pressed[:down]
      @y_velocity = speed;
    else
      @y_velocity = 0;
    end

    super
  end

end
